update_policy.ml

```diff
@@ -0,0 +1,517 @@
+let update_policy_layout (user : User_model.user) ~user_policy ~root_policy
+    ~unallocated_resources =
+  Tyxml_html.(
+    section
+      ~a:[ a_id "policy-form" ]
+      [
+        h2
+          ~a:[ a_class [ "font-semibold text-xl" ] ]
+          [ txt ("Set Policy for " ^ user.name) ];
+        p ~a:[ a_id "form-alert"; a_class [ "my-4" ] ] [];
+        p ~a:[ a_id "user_id"; a_class [ "hidden" ] ] [ txt user.uuid ];
+        div
+          ~a:[ a_class [ "my-3" ] ]
+          [
+            label
+              ~a:[ a_class [ "block text-sm font-medium" ] ]
+              [ txt "Allowed unikernels" ];
+            p
+              ~a:[ a_class [ "space-x-5" ] ]
+              [
+                txt
+                  ("total available: "
+                  ^ string_of_int
+                      Vmm_core.Policy.(root_policy.vms - user_policy.vms));
```

Comment:
```suggestion
                      Vmm_core.Policy.(root_policy.vms + user_policy.vms));
```

in our discussion, root_policy.vms would be 3, user_policy.vms for robur 5, and we want to display 8 -- not "-2".